tools/WixBuild.vcxproj.props

```diff
@@ -52,6 +53,8 @@
   <PropertyGroup>
     <ResourceOnlyDll Condition=" '$(ResourceOnlyDll)'=='' ">false</ResourceOnlyDll>
     <GenerateManifest Condition="$(ResourceOnlyDll)">false</GenerateManifest>
+    <WindowsSDKDesktopARMSupport Condition=" '$(Platform)'=='arm' and $(VS2017Available) ">true</WindowsSDKDesktopARMSupport>
```

Comment:
I didn't auto enable building ARM, so you still have to add /p:BuildARM=true to the build command line. For this property, I should have replaced `VS2017Available` with `Exists(PlatformSdkArmLibraryRoot)` because it's about whether the WinSDK has ARM and not the build tools. I think we have to assume that the person installs the ARM support in VS for now, just like we do for XP support.